runtime/karura/src/xcm_config.rs

```diff
@@ -287,6 +287,16 @@ parameter_types! {
 	pub const MaxAssetsForTransfer: usize = 2;
 }
 
+parameter_type_with_key! {
+	pub ParachainMinFee: |location: MultiLocation| -> u128 {
+		#[allow(clippy::match_ref_pats)] // false positive
+		match (location.parents, location.first_interior()) {
+			(1, Some(Parachain(parachains::statemine::ID))) => 4_000_000_000,
```

Comment:
this should be managed by XcmInterface pallet so that we can dynamically updating it if needed